torch/utils/mobile_optimizer.py

```diff
@@ -4,7 +4,7 @@
 
 import torch
 from enum import Enum
-from torch._C import MobileOptimizerType
+from torch._C import _MobileOptimizerType as MobileOptimizerType
```

Comment:
You can update https://github.com/pytorch/pytorch/blob/master/torch/_C/__init__.pyi.in#L172-L174 to appease the lint